spec/factories/pathways.rb

```diff
@@ -2,6 +2,6 @@
   factory :pathway do
     title { "Pathway One" }
     details { "Lorem ipsum dolor sit amet, officia excepteur ex fugiat reprehenderit enim labore culpa sint ad nisi Lorem pariatur mollit ex esse exercitation amet. Nisi anim cupidatat excepteur officia. Reprehenderit nostrud nostrud ipsum Lorem est aliquip amet voluptate voluptate dolor minim nulla est proident. Nostrud officia pariatur ut officia. Sit irure elit esse ea nulla sunt ex occaecat reprehenderit commodo officia dolor Lorem duis laboris cupidatat officia voluptate. Culpa proident adipisicing id nulla nisi laboris ex in Lorem sunt duis officia eiusmod. Aliqua reprehenderit commodo ex non excepteur duis sunt velit enim. Voluptate laboris sint cupidatat ullamco ut ea consectetur et est culpa et culpa duis." }
-    dificulty { :beginer } 
+    difficulty { :beginer } 
```

Comment:
not relevant to this, but we might want to switch difficult to enums, or at least validate pathway into one of three choices, beginner, intermediate, advanced for example, in which case check spelling 😉

Reply:
Could you expand on the set output and node.js 16 stuff. or make a ticket for those fixes.